java-client-api-tutorial/step1.md

```diff
@@ -0,0 +1,30 @@
+# Welcome to: Using the Java client library to access the Conjur API!
+
+Welcome to Conjur's tutorial of how to use our Java client library to access our Conjur API.
+
+In this tutorial, you will learn how to run a local Java application that connects to the Conjur API given an access token provided by a Kubernetes authenticator sidecar on a local Kubernetes cluster using Minikube, login to the Conjur CLI, authenticate and set policies, and fetch a secret.
+
+## Overview
+Applications and application developers should be incapable of leaking secrets.
+
+In order to achieve this goal, you will play three roles in this tutorial:
+1. An **Infrastructure Engineer**, who will deploy an instance of Conjur
+2. A **Security Admin** who handles secrets, and has sole access to their secrets
+3. An **Application Developer** who wants to connect to Conjur to access secrets to be used in their application.
```

Comment:
```suggestion
3. An **Application Developer** who wants to deploy their Java application to Kubernetes, and make sure it has the credentials to connect to the resources it needs
```
you've been an app developer. when you're building your app, is your primary goal ever "connecting to a secrets service"?